common/djangoapps/monkey_patch/third_party_auth.py

```diff
@@ -26,3 +29,14 @@ def _create_social_auth(*args, **kwargs):
         return classmethod(_create_social_auth)
 
     DjangoUserMixin.create_social_auth = create_social_auth_wrapper(DjangoUserMixin.create_social_auth)
+
+
+def patch_cms():
```

Comment:
@doctoryes Since python-social-auth is used in both LMS and CMS shouldn't we patch all these things for both of them?